std/typecons.d

```diff
@@ -535,14 +535,14 @@ if (distinctFieldNames!(Specs))
     string injectNamedFields()
     {
         string decl = "";
+        import std.conv : to;
         foreach (i, name; staticMap!(extractName, fieldSpecs))
         {
-            import std.format : format;
```

Comment:
Wow. Yeah, `std.format` is a bear in CTFE.  It's pretty awesome that you can use it in CTFE, but performance is thrown out the window.